src/privatesend.h

```diff
@@ -204,11 +199,6 @@ class CDarksendQueue
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action) {
         READWRITE(nDenom);
-        int nVersion = s.GetVersion();
```

Comment:
@UdjinM6 Will this change the wire serialization of DSQ (and cause incompatibility between versions)? Or does that not matter since Spork 6 was never activated?

Reply:
Yes, serialization will change, but we are bumping `MIN_PRIVATESEND_PEER_PROTO_VERSION` too here (and we don't process PS messages from/to to nodes below it), so it should be fine.

Reply:
All 12.4 nodes will reject DSA and DSQ from any 12.3 node, and all 12.3 nodes will not be able to decode DSA and DSQ from any 12.4 node.

I see 2 issues here during the network upgrade:
1) there may be problems with propagation of DSQ messages through the network,
2) 12.3 nodes can mix only with another 12.3 nodes only through 12.3 MNs, the same for 12.4 nodes/MNs

The Second issue would not be so big if the First issue didn't make mixing unstable for quite long time. Thoughts?

Reply:
Small correction: 12.4 won't relay any PS message to 12.3 nodes. But yes, the migration issues you highlighted are correct - mixing usability will be degraded for a week or so while network is upgrading, especially if you are not using the version which is used by the majority (MNs/mixing participants) atm.